specs/validations.md

```diff
@@ -42,8 +48,16 @@ submits it to the `L2OutputOracle` contract on L1.
 
 [finality]: https://hackmd.io/@prysmaticlabs/finality
 
-The submission of output roots is permitted to a single account. It is expected that this
-account continues to submit output roots over time to ensure that user withdrawals do not halt.
+The validator that submits the output root is determined by the `ValidatorPool` contract on L1.
+The output submission rounds are `Priority Round` and `Public Round`, and the time limit of each round is configured
+as `ROUND_DURATION` in the ValidatorPool contract.
+A prioritized validator is selected by a random function in the `ValidatorPool` contract, and the prioritized validator
+must submit output within the Priority Round time.
+If the prioritized validator fails to submit within the Priority Round, the round moves to the Public Round, where all
+validators can submit output regardless of priority.
+
+Once your submitted output is [finalized][finality], the submitter is eligible for a reward.
+For more information on this, see [Validation Rewards](#validation-rewards).
```

Comment:
I think we should add checking next validator code block to L72-L82. Also remove L84-L85.